tests/views/test_message.py

```diff
@@ -0,0 +1,38 @@
+import json
+import unittest
+from unittest.mock import MagicMock
+
+import requests_mock
+from requests import RequestException
+
+from config import TestingConfig
+from response_operations_ui import app
+
+get_message_list = f'{app.config["BACKSTAGE_API_URL"]}/v1/secure-message/messages'
+with open('tests/test_data/survey/survey_list.json') as json_data:
+    message_list = json.load(json_data)
+
+
+class TestMessage(unittest.TestCase):
+
+    def setUp(self):
+        app_config = TestingConfig()
```

Comment:
Not sure this is needed as this will run as part of run_tests which is set up to use testing config.

Reply:
Fixed